### PR TITLE
fix: fixed prop types, logout, fixed alerts in login page

### PR DESCRIPTION
**Pull request by Marina20V**
dev
## GitHub Project

* [Main GitHub Project ticket](https://github.com/ita-social-projects/EventsExpress-Frontend/projects/1)

## Code reviewers

- [ ] @VadymKharyton 
- [ ] @andrii-kuzmichov 

### Second Level Review

- [ ] @github_username

## Summary of issue

Fix login in the page

## Summary of change

fixed prop types, logout, fixed alerts.

## Testing approach

ToDo

## CHECK LIST
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/containers/FacebookLogin.js [hunk 1]
```diff
@@ -3,15 +3,15 @@ import PropTypes from "prop-types";
...
 import "./css/Auth.css";
 
 const LoginFacebook = ({ config }, props) => {
   const responseFacebook = response => {
-    //! TODO: LOGIN ISN'T USED ANYWHERE
-    // if (typeof response.email === "undefined") {
-    // let loginCopy = props.login.loginError;
-    // loginCopy = "Please add email to your facebook account!";
-    // }
+    if (typeof response.email === "undefined") {
```
Comment (VadymKharyton) [inline src/containers/FacebookLogin.js]: Simplify to `if (!response.email)`

File: src/containers/GoogleLogin.js [hunk 2]
```diff
@@ -9,10 +9,9 @@ import "./css/Auth.css";
 
 class LoginGoogle extends Component {
   googleResponseHandler = response => {
-    //! TODO: LOGIN ISN'T USED ANYWHERE
-    // if (typeof response.profileObj.email === "undefined") {
-    //   this.props.login.loginError = "Please add email to your google account!";
-    // }
+    if (typeof response.profileObj.email === "undefined") {
```
Comment (VadymKharyton) [inline src/containers/GoogleLogin.js]: Similar condition, please simplify

Comment (Marina20V) [reply src/containers/GoogleLogin.js]: Done

File: src/containers/FacebookLogin.js (hunk 1 above)
Comment (Marina20V) [reply src/containers/FacebookLogin.js]: Done

**Review by VadymKharyton — COMMENTED** [review]
There are some errors during PR build. I created a file with build logs.
[build-issues.txt](https://github.com/ita-social-projects/EventsExpress-Frontend/files/8018407/build-issues.txt)

**Comment by Marina20V** [comment]
> There are some errors during PR build. I created a file with build logs. [build-issues.txt](https://github.com/ita-social-projects/EventsExpress-Frontend/files/8018407/build-issues.txt)

need approve

**Review by VadymKharyton — APPROVED** [review]

**Pull request merged**